yb_migrate/cmd/analyzeSchema.go

```diff
@@ -784,6 +784,16 @@ func analyzeSchema() {
 		utils.ErrExit("failed to write report to %q: %s", reportPath, err)
 	}
 	fmt.Printf("-- find schema analysis report at: %s\n", reportPath)
+
+	//add diagnostic flag check here...
+	utils.InitJSON(exportDir)
```

Comment:
instead of using utils package, create a new package `callhome` or `diagnostics` where all the related function are placed.
You can refer to src or target packages or [datafile package](https://github.com/yugabyte/yb-voyager/pull/143) in my current PR

Reply:
It will also make code more readable currently utils doesn't signify anything. diagnostics or callhome will be more meaningful

Reply:
Done